src/pages/journal/journal-components/journal-time/journal-time.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'journal-time',
+  templateUrl: 'journal-time.html',
+  inputs: ['time','testComplete']
```

Comment:
Don't think you need `inputs` in the `@Component` decorator - I think the `@Input` decorator is enough.

Reply:
Done